app/views/users/piv_cac_setup_from_sign_in/prompt.html.erb

```diff
@@ -25,11 +25,16 @@
   <div class="margin-top-5">
     <%= f.submit t('forms.piv_cac_setup.submit') %>
   </div>
-  <div class="margin-top-2">
-    <%= link_to t('forms.piv_cac_setup.no_thanks'),
-                new_user_session_url,
-                class: 'usa-button usa-button--wide usa-button--big usa-button--outline' %>
-  </div>
 <% end %>
 
+<%= render ButtonComponent.new(
+      action: ->(**tag_options, &block) do
+        button_to(login_add_piv_cac_prompt_path, **tag_options, method: :post, &block)
```

Comment:
If I log in without an SP and click this, it takes me to the same page. Is that intended?

Reply:
> If I log in without an SP and click this, it takes me to the same page. Is that intended?

Hm, no, I would expect not. I was relying on the existing behavior of `Users::PivCacSetupFromSignInController#decline`, but it sounds like maybe there's a bug there. I'll check it out.

Reply:
@mitchellhenke I wasn't able to reproduce this, would you be able to write up some specific steps to reproduce?

Reply:
I can add an additional feature test case for this though, since the current one only covers sign in with partner.

Reply:
Added coverage in fa8144d72. Seems to be passing as-is for me.

Reply:
Hm, I can't reproduce anymore either, fine to ignore.